doc/api/v8.md

```diff
@@ -520,5 +547,6 @@ A subclass of [`Deserializer`][] corresponding to the format written by
 [`serializer.releaseBuffer()`]: #v8_serializer_releasebuffer
 [`serializer.transferArrayBuffer()`]: #v8_serializer_transferarraybuffer_id_arraybuffer
 [`serializer.writeRawBytes()`]: #v8_serializer_writerawbytes_buffer
+[`v8.stopCoverage()`]: #v8_stop_coverage
```

Comment:
```suggestion
[`v8.stopCoverage()`]: #v8_v8_stopcoverage
```

Reply:
I am not sure how the grammar rule matches. But I look at the generated html. This should fix the doc lint error.

Reply:
Fixed. Thanks!

Reply:
@joyeecheung It's `v8_v8_stopcoverage` there is not _ in stop and coverage.
![image](https://user-images.githubusercontent.com/3759816/96541684-1be00e00-12d3-11eb-8d62-3ac284418779.png)